console/console-init/ui/src/components/BulkSelect/BulkSelect.tsx

```diff
@@ -24,7 +23,6 @@ export const BulkSelect: React.FunctionComponent<IBulkSelectProps> = ({
   isChecked,
```

Comment:
not sure about the purpose this component but i can see that you are using PF Dropdown component. can we try component name something else like  DropdownWithBulkSelect.

Reply:
This is for [BulkSelect](https://www.patternfly.org/v4/design-guidelines/usage-and-behavior/bulk-selection#integrated-bulk-selection-for-tables).